turbopack/crates/turbopack-browser/src/chunking_context.rs

```diff
@@ -94,6 +95,11 @@ impl BrowserChunkingContextBuilder {
         self
     }
 
+    pub fn chunking_strategy(mut self, chunking_strategy: Vc<Box<dyn ChunkingStrategy>>) -> Self {
```

Comment:
If these strategies are only ever implemented in the turbopack-core crate and never by users (e.g. Next), maybe this should just be an enum instead?

Reply:
I think it's fine either way. It does seem unlikely that we would need to let frameworks (next) declare custom chunking strategies.